index.js

```diff
@@ -27,11 +27,20 @@ const buildQueryParams = function(vin, scope, country, options) {
   if (options.flags) {
     parameters.flags = util.getFlagsString(options.flags);
   }
-
   if (options.hasOwnProperty('testMode')) {
-    parameters.mode = options.testMode ? 'test' : 'live';
+    // eslint-disable-next-line no-console
+    console.warn(
```

Comment:
Use [`process.emitWarning`](https://nodejs.org/api/process.html#processemitwarningwarning-options) instead of `console.warn`.  `console.warn` would be a good choice if our SDK targeted browsers and Node.js, but since we only target Node.js runtimes, `process.emitWarning` is a better fit.